pkgs/README.md

```diff
@@ -521,6 +521,9 @@ When using the `patches` parameter to `mkDerivation`, make sure the patch name c
 >
 > See [Versioning](#versioning) for details on package versioning.
 
+No matter which method you take to include the patch, you _must_ state the purpose of the patch clearly, either by adding a comment above it, or by setting the filename of the patch.
+A good explanation _should_ at least include "what did it fix" and "how did it fix", for example "fix gcc14 build by adding missing includes". However, if the fix is directly reflected in the code and requires no further explanation, the "how" part _shall_ be omitted.
```

Comment:
the last sentences kinda contradicts the previous two, tldr: "please explain except when it is self-explanatory". I would remove it or rephrase the previous one to make the "how" optional